grammar.js

```diff
@@ -988,15 +988,16 @@ module.exports = grammar({
         ),
         
 
+        // https://github.com/tree-sitter/tree-sitter-c/blob/master/grammar.js#L965
         comment: $ => token(
-            prec(PREC.COMMENT, 
+            prec(PREC.COMMENT,
                 choice(
-                    seq('//', /.*/),
+                    seq('//', /(\\(.|\r?\n)|[^\\\n])*/),
                     seq(
                         '/*',
-                        /.*/,
-                        '*/'
-                    )       
+                        /[^*]*\*+([^/*][^*]*\*+)*/,
+                        '/'
```

Comment:
Isn't it required that the docstring ends with '*/' without any character in between?
```suggestion
                        '*/'
```

Reply:
I'm not sure, do what you feel is best, I just copied it from the C version and assumed it was correct.